app/decorators/app_decorator.rb

```diff
@@ -7,6 +7,18 @@ def email_at_notices
     object.email_at_notices.join(', ')
   end
 
+  def use_site_fingerprinter
```

Comment:
I find it easier to read when I break these down onto multiple lines of guard clauses:
```ruby
def use_site_fingerprinter
  return true if object.notice_fingerprinter.nil?
  return true if object.notice_fingerprinter.attributes['source'].nil?
  object.notice_fingerprinter.attributes['source'] == SiteConfig::CONFIG_SOURCE_SITE
end